components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/EntityTypeEntity.java

```diff
@@ -0,0 +1,134 @@
+package com.sap.cloud.cmp.ord.service.storage.model;
+
+import com.sap.olingo.jpa.metadata.core.edm.annotation.EdmIgnore;
+import com.sap.olingo.jpa.metadata.core.edm.annotation.EdmProtectedBy;
+import org.eclipse.persistence.annotations.Convert;
+import org.eclipse.persistence.annotations.TypeConverter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+@Entity(name = "entityType")
+@Table(name = "tenants_entity_types")
+public class EntityTypeEntity {
```

Comment:
Add

`    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "package_id", insertable = false, updatable = false)
    private PackageEntity pkg;`
    
    so that we can expand the pkg in which the entityType is part of in the odata query.